projects/organization-management/src/app/pages/hierarchies-create-group/hierarchies-create-group-page.component.ts

```diff
@@ -0,0 +1,68 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { UUID } from 'angular2-uuid';
+import { Observable } from 'rxjs';
+
+import { HttpError } from 'ish-core/models/http-error/http-error.model';
+import { markAsDirtyRecursive } from 'ish-shared/forms/utils/form-utils';
+
+import { OrganizationManagementFacade } from '../../facades/organization-management.facade';
+import { Node, NodeTree } from '../../models/node/node.model';
+
+@Component({
+  selector: 'ish-hierarchies-create-group-page',
+  templateUrl: './hierarchies-create-group-page.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class HierarchiesCreateGroupPageComponent implements OnInit {
+  form: FormGroup = this.fb.group({
+    org_group: this.fb.group({
```

Comment:
Just a matter of taste: do you actually need the underscore here? We are accustomed to camel-casing. 🐫

Reply:
Nah. I can change that.